frontend/packages/topology/src/components/dropdowns/ApplicationDropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { Firehose } from '@console/internal/components/utils';
+import { ResourceDropdown } from '@console/shared';
+import { useExtensions } from '@console/plugin-sdk/src';
+import { isTopologyDataModelFactory, TopologyDataModelFactory } from '../../extensions';
+import { WatchK8sResources } from '@console/internal/components/utils/k8s-watch-hook';
+import { getBaseWatchedResources } from '../../data-transforms';
+
+interface ApplicationDropdownProps {
+  id?: string;
+  className?: string;
+  dropDownClassName?: string;
+  menuClassName?: string;
+  buttonClassName?: string;
+  title?: React.ReactNode;
+  titlePrefix?: string;
+  allApplicationsKey?: string;
+  storageKey?: string;
+  disabled?: boolean;
+  allSelectorItem?: {
+    allSelectorKey?: string;
+    allSelectorTitle?: string;
+  };
+  noneSelectorItem?: {
+    noneSelectorKey?: string;
+    noneSelectorTitle?: string;
+  };
+  namespace?: string;
+  actionItems?: {
+    actionTitle: string;
+    actionKey: string;
+  }[];
+  selectedKey: string;
+  autoSelect?: boolean;
+  onChange?: (key: string, name?: string) => void;
+  onLoad?: (items: { [key: string]: string }) => void;
+}
+
+export const ApplicationDropdown: React.FC<ApplicationDropdownProps> = ({
```

Comment:
Default export?

Reply:
Trying to make the topology components consumable from `@console/topology` so we need to either export explicitly here or in the index file.

Reply:
I don't believe making it an explicit export makes it any different than a default export except for the notation used to import it. 

Are you experiencing a specific issue with default exports?

Reply:
I just dislike:
```
export { default as ApplicationDropdown } from './ApplicationDropdown;
```